jina/drivers/search.py

```diff
@@ -127,9 +127,8 @@ def _apply_all(self, docs: 'DocumentSet', *args, **kwargs) -> None:
         for doc, topks, scores in zip(doc_pts, idx, dist):
 
             topk_embed = fill_fn(topks) if (self._fill_embedding and fill_fn) else [None] * len(topks)
-
             for match_hash, score, vec in zip(topks, scores, topk_embed):
-                r = doc.add_match(doc_id=match_hash,
+                r = doc.add_match(doc_id=uid.hash2id(match_hash),
```

Comment:
i remember I add shortcut for hash-value `doc_id` in `add_match`, let me check

Reply:
![image](https://user-images.githubusercontent.com/2041322/99874506-d58b1100-2be8-11eb-9aad-b829c661de47.png)
yes,  I think this is not needed